keanu-python/setup.py

```diff
@@ -1,18 +1,26 @@
-from setuptools import setup
+from setuptools import setup, find_packages
+import shutil
 
 def readme():
     with open('README.rst', "r") as f:
         return f.read()
 
+version_string = '0.0.15.dev0'
+
+# If you don't remove the old directories, it tends to put the excluded module "examples" into the bdist
+for dir_name in ("keanu-%s.dist-info" % version_string, "keanu.egg-info", "build", "dist"):
```

Comment:
It seems like we can get around this bug my adding some more stuff to our `MANIFEST.in`

[Source](https://stackoverflow.com/questions/26545668/setup-py-sdist-exclude-packages-in-subdirectory)

Reply:
I don't think so - see e.g. https://github.com/pypa/setuptools/issues/511